api/tools/bots/index.ts

```
@@ -0,0 +1,10 @@
+import { BaasClient } from "@meeting-baas/sdk";
+import { McpServer } from "@modelcontextprotocol/sdk/server/mcp.js";
+import { registerJoinTool } from "./join";
+
+export function registerBotTools(server: McpServer, baasClient: BaasClient): McpServer {
+  // Register all bot-related tools
+  server = registerJoinTool(server, baasClient);
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)